src/test/java/org/gridsuite/study/server/StudyTest.java

```diff
@@ -635,6 +635,10 @@ public MockResponse dispatch(RecordedRequest request) {
                         return new MockResponse().setResponseCode(200).addHeader("Content-Disposition", "attachment; filename=fileName").setBody("byteData")
                             .addHeader("Content-Type", "application/json; charset=utf-8");
 
+                    case "/v1/networks/" + NETWORK_UUID_STRING + "/export/XIIDM" + "?variantId=variant_1":
```

Comment:
Maybe use existing constant field VARIANT_ID instead of "variant_1"

Reply:
done